Sources/App/Views/Supporters/SupportersShow+Model.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Plot
+
+enum SupportersShow {
+    struct Model {
+        func corporateSupporters() -> Node<HTML.BodyContext> {
+            .section(
+                .class("corporate"),
+                .h3("Corporate Supporters"),
+                .ul(
+                    .group(
+                        Supporters.corporate.shuffled().map(\.listNode)
```

Comment:
We probably need to use `Current` here to control randomness in snapshots, or am I missing how that’s controlled?

Reply:
The mock repeats the same data repeated, which tests the layout without difficulties caused by `shuffled` and also without including real profiles in our tests.